### PR TITLE
[HTTPS] language files

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

This PR changes all http:// URI in language ini files that use sites exclusive in HTTPS to https://
#### Testing Instructions

Check code difference.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha.sys.ini [hunk 1]
```diff
@@ -3,5 +3,5 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION="This CAPTCHA plugin uses the reCAPTCHA service to prevent spammers while it helps to digitize books, newspapers and old radio shows. To get a site and secret key for your domain, go to http://www.google.com/recaptcha. To use this for new account registration, go to Options in the User Manager and select Captcha - reCaptcha as the Captcha."
+PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION="This CAPTCHA plugin uses the reCAPTCHA service to prevent spammers while it helps to digitize books, newspapers and old radio shows. To get a site and secret key for your domain, go to <a href='https://www.google.com/recaptcha' target='_blank'>https://www.google.com/recaptcha</a>. To use this for new account registration, go to Options in the User Manager and select Captcha - reCaptcha as the Captcha."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha.sys.ini]: As you have changed this to a link please use _QQ_ 
eg <a href="_QQ_"https://www.google.com/recaptcha"_QQ_" target="_QQ_"_blank"_QQ_">

File: administrator/language/en-GB/en-GB.plg_editors_codemirror.ini [hunk 2]
```diff
@@ -15,7 +15,7 @@ PLG_CODEMIRROR_FIELD_AUTOFOCUS_DESC="Auto focus."
 PLG_CODEMIRROR_FIELD_AUTOFOCUS_LABEL="Auto Focus"
 PLG_CODEMIRROR_FIELD_CODEFOLDING_DESC="Allow blocks of code to be folded."
 PLG_CODEMIRROR_FIELD_CODEFOLDING_LABEL="Code Folding"
-PLG_CODEMIRROR_FIELD_FONT_FAMILY_DESC="The font to use in the editor. If not installed, will be loaded from http://www.google.com/webfonts."
+PLG_CODEMIRROR_FIELD_FONT_FAMILY_DESC="The font to use in the editor. If not installed, will be loaded from <a href='https://www.google.com/fonts/' target='_blank'>https://www.google.com/fonts/</a>."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_editors_codemirror.ini]: _QQ_

File: administrator/language/en-GB/en-GB.plg_system_languagecode.ini [hunk 3]
```diff
@@ -7,4 +7,4 @@ PLG_SYSTEM_LANGUAGECODE="System - Language Code"
 PLG_SYSTEM_LANGUAGECODE_FIELD_DESC="Changes the language code used for the <em>%s</em> language."
 PLG_SYSTEM_LANGUAGECODE_FIELDSET_DESC="Changes the language code for the generated HTML document. Example of use: You have installed the fr-FR language pack and want the Search Engines to recognise the page as aimed at French-speaking Canada. Add the tag 'fr-CA' to the corresponding field for 'fr-FR' to resolve this."
 PLG_SYSTEM_LANGUAGECODE_FIELDSET_LABEL="Language Codes"
-PLG_SYSTEM_LANGUAGECODE_XML_DESCRIPTION="Provides the ability to change the language code in the generated HTML document to improve SEO.<br />The fields will appear when the plugin is enabled and saved.<br />More information at <a href="_QQ_"http://www.w3.org/TR/xhtml1/#docconf"_QQ_">W3.org</a>."
+PLG_SYSTEM_LANGUAGECODE_XML_DESCRIPTION="Provides the ability to change the language code in the generated HTML document to improve SEO.<br />The fields will appear when the plugin is enabled and saved.<br />More information at <a href='https://www.w3.org/TR/xhtml1/#docconf' target='_blank'>W3.org</a>."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_system_languagecode.ini]: _QQ_

**Comment by brianteeman** [comment]
Not going to comment everywhere but please use _QQ_ for links eg
`< a href="_QQ_"https://www.google.com/recaptcha"_QQ_" target="_QQ_"_blank"_QQ_">`

**Comment by brianteeman** [comment]
Also you have changed some urls in tooltips to links. No point in doing that as you cannot click on a tooltip ;)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9386">issues.joomla.org/joomla-cms/9386</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@brianteeman what tooltips?
Also, why use `"_QQ_"` instead of just `'`. Can you explain me the reason for that? Just asking to understand.

**Comment by brianteeman** [comment]
If a string ends _DESC and is not a .sys.ini file then it is a tooltip

Really dont want to go into the long stupid story about _QQ_ but you need to do that

**Comment by mbabker** [comment]
Didn't we agree already to deprecate `"_QQ_"` or am I back in wishful thinking land?

**Comment by brianteeman** [comment]
@mbabker we did but then it broke com_localise and I lost the will to argue and dont have it back

**Comment by andrepereiradasilva** [comment]
ok i will put the QQ, when deprecated we can remove all of then at once.
ok for the tooptip (there is one i changed)

**Comment by mbabker** [comment]
It only breaks if you don't have the slightest clue how to correctly escape quotes in strings.  But alas.  Removing it one day is a B/C break, and heaven forbid anyone EVER break B/C short of a security issue (I've been reading too much PHP internals lately).

**Comment by brianteeman** [comment]
Lets not get distracted by useless conversations ;)

On 12 March 2016 at 17:06, Michael Babker <notifications@github.com> wrote:

> It only breaks if you don't have the slightest clue how to correctly
> escape quotes in strings. But alas. Removing it one day is a B/C break, and
> heaven forbid anyone EVER break B/C short of a security issue (I've been
> reading too much PHP internals lately).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/9386#issuecomment-195775517>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by andrepereiradasilva** [comment]
O_qq_ now?

**Comment by brianteeman** [comment]
I wish you had just changed the protocol and didnt change anything else but you have added target=blank to everything which is considered to be bad practice for accessibility and turned things that were not links into links - I assume there was a reason that they were not links before so now need to be reviewed

**Comment by andrepereiradasilva** [comment]
ok i will remove the target=_blank...

reggarding the links i just added then in the XML description... that i know of there is no problem with that.

**Comment by andrepereiradasilva** [comment]
@brianteeman can/should the *.sys.ini files have links?
If not i will remove the ones i added in those files.

**Comment by brianteeman** [comment]
Yes but I'm not sure if they were not made as links for a specific reason
or just an errror

**Comment by andrepereiradasilva** [comment]
here we have target_blank and links in a sys.ini ...
https://github.com/joomla/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.plg_authentication_gmail.sys.ini#L7

**Comment by infograf768** [comment]
I would not touch the targets at all or clickable/non clickable nature of the urls. 
The only change imho should be the http =>https when necessary.

**Comment by andrepereiradasilva** [comment]
ok. will remove all changes except the https ones.

**Comment by andrepereiradasilva** [comment]
ok. now only http to https please check

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e9c2680cebca4b3f7508b17d9d38e907e6dcffab<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9386">issues.joomla.org/joomla-cms/9386</a>.</sub>

**Comment by brianteeman** [comment]
Thank you<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9386">issues.joomla.org/joomla-cms/9386</a>.</sub>

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on e9c2680cebca4b3f7508b17d9d38e907e6dcffab<br /><br />simple fix. thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9386">issues.joomla.org/joomla-cms/9386</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9386">issues.joomla.org/joomla-cms/9386</a>.</sub>

**Pull request merged**